routing/routing.go

```diff
@@ -369,8 +386,17 @@ func (r *Routing) startReceivingUpdates(o Options) {
 						r.firstLoadSignaled = true
 					}
 				}
+
+				close(goodEndpointerQuit)
+				timer.Stop()
+				goodEndpointerQuit = make(chan struct{})
+				timer = time.NewTicker(500 * time.Millisecond)
+				go GoodEndpointerGoroutine(rt.routes, goodEndpointerQuit, timer.C)
```

Comment:
this code block feels if we could make it better, but I have no idea how. On the other hand it's only run every time on routing tree change, so maybe it's fine. I am a bit worried about high traffic while updating this here.
@AlexanderYastrebov wdyt?